### PR TITLE
Fix GH-2357: improve `Traversable` page

### PR DESCRIPTION
**Pull request by alexandre-daubois**
Fix https://github.com/php/doc-en/issues/2357

### CODE REVIEW

File: language/predefined/traversable.xml [hunk 1]
```diff
@@ -58,6 +39,54 @@
...
+     <interfacename>IteratorAggregate</interfacename> or
+     <interfacename>Iterator</interfacename>.
+    </para>
+   </note>
+   <note>
+    <para>
+     Prior to PHP 8.0.0, this internal engine interface couldn't be implemented
+     in PHP scripts. Either <interfacename>IteratorAggregate</interfacename>
+     or <interfacename>Iterator</interfacename> must be used instead.
+     When implementing an interface which extends Traversable, make sure to
```
Comment (Girgias) [inline language/predefined/traversable.xml:82]: Yes this was already missing but might as well add it

```suggestion
     When implementing an interface which extends <interfacename>Traversable</interfacename>,
     make sure to
```

Comment (Girgias) [reply language/predefined/traversable.xml:82]: But also is the ordering still important?

**Review by Girgias — APPROVED** [review]
Minor comments but LGTM

File: language/predefined/traversable.xml (hunk 1 above)
Comment (alexandre-daubois) [reply language/predefined/traversable.xml]: I tested and actually no, I updated it :slightly_smiling_face:

Comment (alexandre-daubois) [reply language/predefined/traversable.xml]: Tried on 7.0, here is the result:

image

When switching order:

image

So I'm not exactly sure from when this rule applied

**Comment by alexandre-daubois** [comment]
Yes it seems you're right. Sorry for the brittle check, I updated the XML :slightly_smiling_face: Set the changelog entry to 7.4.0. I'm not sure the ordering rule ever applied :thinking:

**Comment by Girgias** [comment]
> Yes it seems you're right. Sorry for the brittle check, I updated the XML slightly_smiling_face Set the changelog entry to 7.4.0. I'm not sure the ordering rule ever applied thinking

The ordering rule did apply prior to PHP 7.4: https://3v4l.org/ecv2m but that is a different issue all together

**Review by Girgias — APPROVED** [review]

**Pull request merged**